snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -186,6 +224,16 @@ def last_step(self):
 
         return None
 
+    def next_step(self):
```

Comment:
this should probably be queried by the lifecycle runner eventually

Reply:
Ah, yeah that's a good idea, let's keep that in mind for a future refactor.